<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialMainWindow</class>
 <widget class="QMainWindow" name="SerialMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="portNumberComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portNumberLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>65</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="baudRateLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>105</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataBitsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>145</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopBitsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>185</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="parityBitsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>225</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBitsComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="stopBitsComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parityBitsComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portAreaLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port 1</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>30</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>37</y>
      <width>3</width>
      <height>310</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>338</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>37</y>
      <width>3</width>
      <height>310</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="serialConnectButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Connect (F7) 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialDisconnectButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Disconnect (F8) 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="receivedSentAreaLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>25</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Recevied/Sent Data 1</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>30</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>318</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>40</y>
      <width>3</width>
      <height>470</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>500</y>
      <width>830</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>1147</x>
      <y>40</y>
      <width>3</width>
      <height>470</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveDataButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>365</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Data (F9) 1 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchDataButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>405</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Data (Ctrl + F) 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="sendDataAreaLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send Data 1</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="firstSendDataTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="firstSendDataCheckBoxASCII">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>625</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="firstSendDataCheckBoxHEX">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>625</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondSendDataTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>670</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="thirdSendDataTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>720</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionSetLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connection Set 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dtrCheckBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DTR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rtsCheckBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RTS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dsrCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DSR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ctsCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CTS</string>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>469</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>479</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>978</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="firstSendDataSendButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>620</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstSendDataClearButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>620</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>1147</x>
      <y>600</y>
      <width>3</width>
      <height>164</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="secondSendDataCheckBoxASCII">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>675</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="secondSendDataCheckBoxHEX">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>675</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thirdSendDataCheckBoxASCII">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>725</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thirdSendDataCheckBoxHEX">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>725</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondSendDataClearButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>668</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondSendDataSendButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>668</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thirdSendDataSendButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>716</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thirdSendDataClearButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>716</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>755</y>
      <width>1091</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>600</y>
      <width>3</width>
      <height>164</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="noneRadioButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>550</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="crRadioButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lfRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>LF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="crLFRadioButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>550</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CR + LF</string>
    </property>
   </widget>
   <widget class="QLabel" name="packetAreaLabel">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>550</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Packet 1</string>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_20">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>540</y>
      <width>20</width>
      <height>38</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="portAreaLabel_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status 1</string>
    </property>
   </widget>
   <widget class="QListView" name="dataListView">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>60</y>
      <width>791</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dataListViewClearButton">
    <property name="geometry">
     <rect>
      <x>1059</x>
      <y>464</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
